src/main/resources/templates/subscription.html

```diff
@@ -82,7 +82,7 @@ <h3 id="subData">Subscription Data</h3>
                         <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                     </div>
                 </div>
-                <div class="modal-body form" align="left">
+                <div class="modal-body form" allign="left">
```

Comment:
Misspelled? Should be "align"

Reply:
Should be, but was marked red in Vs code, and OP with allign, think it does not matter so maybe to remove it.